tests/slow_tests/slow_tests.cpp

```diff
@@ -1116,9 +1117,9 @@ R"(
       trx.messages.resize(1);
       trx.messages[0].code = config::EosContractName;
       trx.messages[0].authorization.emplace_back(types::AccountPermission{"simplecoin","active"});
-      trx.setMessage(0, "setcode", handler);
+      transaction_helpers::set_message(trx, 0, "setcode", handler);
```

Comment:
perhaps renaming the methods .  transaction_set_message rather than transaction_helpers::set_message if we are going to be verbose everywhere.  If follows the naming convention one would use for a object-oriented C api.